spec/latest/json-ld/index.html

```diff
@@ -45,27 +45,30 @@
       // editors, add as many as you like
       // only "name" is required
       editors:  [
-          { name: "Manu Sporny", url: "http://manu.sporny.org/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/" },
-          { name: "Gregg Kellogg", url: "http://greggkellogg.net/",
-            company: "Kellogg Associates", companyURL: "http://kellogg-assoc.com/" },
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
-            company: "Graz University of Technology", companyURL: "http://www.tugraz.at/" }
-      ],
-
-      // authors, add as many as you like.
-      // This is optional, uncomment if you have authors as well as editors.
-      // only "name" is required. Same format as editors.
-      authors:  [
-          { name: "Manu Sporny", url: "http://digitalbazaar.com/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/" },
-          { name: "Dave Longley", url: "http://digitalbazaar.com/",
-            company: "Digital Bazaar", companyURL: "http://digitalbazaar.com/"},
-          { name: "Gregg Kellogg", url: "http://greggkellogg.net/",
-            company: "Kellogg Associates", companyURL: "http://kellogg-assoc.com/" },
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
-            company: "Graz University of Technology", companyURL: "http://www.tugraz.at/" },
-          { name: "Niklas Lindström", url: "http://neverspace.net/" }
+        { name:       "Gregg Kellogg",
```

Comment:
This is the one that I find a bit troubling. To the uninitiated, this looks like @gkellogg was the lead designer and editor for JSON-LD. I think it's true for 1.1, but not for 1.0. I don't think this is what  you mean to imply, @gkellogg, but those are the optics.

I'm going to work on a separate PR that does this for 1.1 Syntax:

* JSON-LD 1.1 Editors: Gregg Kellogg (JSON-LD 1.0 and 1.1)
* JSON-LD 1.1 Lead Designers: Manu Sporny (JSON-LD 1.0), Dave Longley (JSON-LD 1.0), Gregg Kellogg (JSON-LD 1.0 and 1.1), Markus Lanthaler (JSON-LD 1.0), Niklas Lindstrom (JSON-LD 1.0)
* JSON-LD 1.1 SpecRef Authors: Manu Sporny, Dave Longley, Gregg Kellogg, Markus Lanthaler, Niklas Lindstrom

What I'm trying to do is look at the specs holistically and give credit where credit is due. I don't want to end up where we are today, which is with Dave Longley's name not showing up at all when the specs are referenced.

Are there any objections to the order proposed above?